From 366cce74d2aa2e5753d8787d415b745fd57fda04 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Sat, 21 Sep 2024 19:29:13 +0200
Subject: [PATCH] support: Add valgrind instructions to <support/fuse.h>
Content-type: text/plain; charset=UTF-8

Replacing an outdated comment (namespace setup is now handled by
support_fuse_init).
---
 support/fuse.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/support/fuse.h b/support/fuse.h
index 4c365fbc0c..1c862bedbe 100644
--- a/support/fuse.h
+++ b/support/fuse.h
@@ -16,8 +16,10 @@
    License along with the GNU C Library; if not, see
    <https://www.gnu.org/licenses/>.  */
 
-/* Before using this functionality, use support_enter_mount_namespace
-   to ensure that mounts do not impact the overall system.  */
+/* To run FUSE tests under valgrind, pass the
+   --sim-hints=fuse-compatible option to valgrind.  This option tells
+   valgrind that additional system calls effectively call back into
+   the current program.  */
 
 #ifndef SUPPORT_FUSE_H
 #define SUPPORT_FUSE_H
-- 
2.43.5

